docs/docs/kds-configuration.md

```diff
@@ -207,32 +154,95 @@ kds {
 }
 ```
 
-### Any enclave for a particular product signed using the same key
-This example allows any enclave running any code to access the key as long as it
-is signed with the same signing key, is part of the same product and is running
-on a secure platform.
-
-The combination of a signing key and a product ID allows for unique keys to be
-generated per product, but shared with all versions of enclaves within a single
-product.
+This avoids the denial of service that happens in the more restrictive case above by giving the enclave host time to patch their systems to bring them back to the `SECURE` state.
+The operational implications of enclave updates are not affected however, as access to the derived key still requires that the enclave have a specific code hash.
```

Comment:
This constraint doesn't affect the operational implications of enclave updates because access to the derived key still requires that the enclave has a specific code hash.